recipes/libsndfile/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "libsndfile" %}
+{% set version = "1.0.27" %}
+{% set md5 = "fd1d97c6077f03b5d984d7956ffedb7a" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: http://www.mega-nerd.com/{{ name }}/files/{{ name }}-{{ version }}.tar.gz
+    md5: {{ md5 }}
+
+build:
+    number: 0
+    skip: true  # [win]
+
+requirements:
+  build:
+    - gcc
+    - libflac
+    - libvorbis
+    - libogg
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libsndfile.dylib  # [osx]
+    - test -f $PREFIX/lib/libsndfile.so  # [linux]
+
+about:
+    home: http://www.mega-nerd.com/libsndfile
+    license: LGPL-2.1
+    license_family: LGPL
+    license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Oh, the license is already in the project. Great! Change the license_file line to: `license_file: COPYING` and you don't need to bundle LICENSE with this recipe.